framework_tvb/tvb/adapters/visualizers/brain.py

```diff
@@ -138,14 +138,16 @@ def generate_preview(self, view_model, figure_size=None):
             surface_gid = self.surface_h5.gid.load().hex
             boundary_url = SurfaceURLGenerator.get_url_for_region_boundaries(surface_gid, self.region_map_gid,
                                                                              self.stored_adapter.id)
+            base_adapter_url = SurfaceURLGenerator.get_url_for_data_page_split(surface_gid, self.stored_adapter.id)
```

Comment:
Can this be correct? 
I see method `def read_data_page_split(self, time_series_gid...` thus expecting a TS GID and not a Surface.gid.

Reply:
You solved this, but it is still a problem. This method does not exist on `SurfaceURLGenerator`.

There is one unit-test failing here.  Pls look on all unit-tests when you say a task is complete!